fl4health/clients/basic_client.py

```diff
@@ -326,21 +326,6 @@ def fit(self, parameters: NDArrays, config: Config) -> Tuple[NDArrays, int, Dict
             metrics,
         )
 
-    def evaluate_after_fit(self) -> Tuple[float, Dict[str, Scalar]]:
```

Comment:
This is being torched in favor of just using validate with `pack_losses_with_val_metrics = True`. We were packing the loss here specifically to facilitate FedDG-GA.